spec/features/workflow/drafting_requirements_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+RSpec.feature "Drafting requirements" do
+  scenario do
+    given_there_is_a_document_with_no_content
+    when_i_visit_the_edit_document_page
+    and_save_without_inputting_anything
+    then_i_should_see_an_error_message_regarding_a_missing_title
+
+    when_i_visit_the_edit_document_page
+    and_save_a_title_which_doesnt_meet_the_validations
```

Comment:
Can we make these more specific and split them up?

Reply:
Yep, previously they were split up and specific but due to adding more and more validations, there is a risk of having a realllllllly long test file. But, as there are only a few, your approach makes more sense.